src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
```

Comment:
> I'm not familiar with xunit parallelization of Theories; does it happen automagically or do I need to specify something?

I've never tried.  I think you'd have to change it to say that tests on the same test class could be parallelized, but I'm not 100% sure that's even possible... they'd probably need to be changed to not be theories and not be on the same class.   Might not be worth it (but the fixes to allow them to be parallelizable are still a good idea).